fms_extras/utils/cache/__init__.py

```diff
@@ -0,0 +1,245 @@
+import abc
+import dataclasses
+from typing import List, Optional, Tuple
+
+import torch
+
+
+def compute_position_ids(
```

Comment:
we should probably have a unit test for this function?

Reply:
This is being tested through test_paged now (removed the utility function as it wasn't used)